### DISCUSSION TITLE
Bagaimana Cara Menyembunyikan Skrip yang Berasal dari Ekstensi?

### DISCUSSION THREAD

**Discussion by yuliakbard** [mecha-cms/mecha#185] [Q&A] [answered]
Kang, bagaimana cara menyembunyikan script yang berasal dari extension. misalnya script dari extension view dan comment. saya sudah coba kondisionalkan dibagian extensionnya. contoh:

``` php
<?php namespace x\view\live;
if(!$site->is('mobile')){
$z=\defined("\\DEBUG")&&\DEBUG?'.':'.min.';\Asset::set(__DIR__.\DS.'..'.\DS.'..'.\DS.'lot'.\DS.'asset'.\DS.'js'.\DS.'index'.$z.'js');
}
...
```

saya mencoba mengkondisional bagian ini, tapi scriptnya tetap muncul.

**Comment by taufik-nurrohman** [comment, marked as answer]
Bisa menambahkan perintah seperti ini di berkas `.\lot\layout\index.php` mas, nggak perlu edit berkas ekstensi secara langsung:

~~~ php
$z = defined('DEBUG') && DEBUG ? '.' : '.min.';
if ($site->is('mobile')) {
    Asset::let(LOT . DS . 'x' . DS . 'view' . DS . 'lot' . DS . 'asset' . DS . 'js' . DS . 'index' . $z . 'js');
}
~~~
